nemo/collections/common/tokenizers/en_ja_tokenizers.py

```diff
@@ -25,33 +25,12 @@ class EnJaProcessor:
 
     def __init__(self, lang_id: str):
         self.lang_id = lang_id
-        self.moses_tokenizer = MosesTokenizer(lang=lang_id)
-        self.moses_detokenizer = MosesDetokenizer(lang=lang_id)
-        self.normalizer = MosesPunctNormalizer(
-            lang=lang_id, pre_replace_unicode_punct=True, post_remove_control_chars=True
-        )
 
     def detokenize(self, tokens: List[str]) -> str:
-        """
-        Detokenizes a list of tokens
-        Args:
-            tokens: list of strings as tokens
-        Returns:
-            detokenized Japanese or English string
-        """
-        return self.moses_detokenizer.detokenize(tokens)
+        return ' '.join(tokens)
 
     def tokenize(self, text) -> str:
-        """
-        Tokenizes text using Moses. Returns a string of tokens.
-        """
-        tokens = self.moses_tokenizer.tokenize(text)
-        return ' '.join(tokens)
+        return text
 
     def normalize(self, text) -> str:
-        # Normalization doesn't handle Japanese periods correctly;
-        # '。'becomes '.'.
-        if self.lang_id == 'en':
-            return self.normalizer.normalize(text)
-        else:
-            return text
```

Comment:
Can we get rid of moses style tokenization here?

Reply:
i think so. in the example of this comment, i can see the model is able to produce both types of periods now

Reply:
To follow up on this, I was wondering if instead of just removing all of this content we could keep it as is to still have the BPE way of doing things. We could then use a separate language identifier like en-byte or ja-byte to use this pre/postprocessing.

Reply:
okay, i made a new processor class when dealing with byte-level encoding to preserve the old, default way of doing things